src/main/java/dev/aziz/movies/controllers/MovieController.java

```diff
@@ -0,0 +1,33 @@
+package dev.aziz.movies.controllers;
+
+import dev.aziz.movies.dtos.MovieDto;
+import dev.aziz.movies.dtos.ReducedMovieDto;
+import dev.aziz.movies.services.MovieService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class MovieController {
+
+    private final MovieService movieService;
+
+    @GetMapping("/movies")
+    public List<ReducedMovieDto> getAllMovies() {
```

Comment:
Instead of returning directly a list of an entity, wrap them into ResponseEntity. This way, you can manage the HTTP code and more (you will need it later)